cmake/SSGCommon.cmake

```diff
@@ -255,6 +261,7 @@ endmacro()
 macro(ssg_build_remediations PRODUCT)
     message(STATUS "Scanning for dependencies of ${PRODUCT} fixes (bash, ansible, puppet, anaconda, ignition, kubernetes and blueprint)...")
 
+    _ssg_combine_remediations(${PRODUCT} "${PRODUCT_REMEDIATION_LANGUAGES}")
     _ssg_build_remediations_for_language(${PRODUCT} "${PRODUCT_REMEDIATION_LANGUAGES}")
```

Comment:
The names of `_ssg_combine_remediations` and `combine_remdiations.py` mislead me a bit.
AFAIU the macro and script are selecting or filtering the appropriate remediations.

Same for `_ssg_build_remediations_for_language`, I think this the macro that is actually combining the fixes into a `${LANGUAGE}-fixes.xml`.